lib/diversity/engine.rb

```diff
@@ -31,6 +31,11 @@ def initialize(options = {})
       @debug_level = 0
       @logger = @options[:logger]
 
+      @cache = Moneta.build do
+        use :Expires, expires: 3600
+        adapter :Memory
```

Comment:
I'm fine with this. However, [the moneta docs](https://github.com/minad/moneta#supported-backends) says that the LRUHash backend is preferrable over the Memory backend.